moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -117,38 +117,54 @@ const ProjectActivityLog = () => {
    * @param {string} field - The field name (column name)
    * @return {boolean} - True if the field is contained in the ProjectActivityLogGenericDescriptions object
    */
-  const isFieldGeneric = field =>
+  const isFieldGeneric = (field) =>
     field in ProjectActivityLogGenericDescriptions;
 
   /**
-   * Makes sure the creation of the project always comes first
+   * checks if event is from new schema and if so finds the updated difference and includes that
+   * in the object
    * @param {Array} eventList - The data object as provided by apollo
-   * @return {Array}
+   * @returns {Array}
    */
-  const reorderCreationEvent = eventList => {
-    // Clone eventList array so it can be mutated
-    let outputList = [...eventList];
+  const getDiffs = (eventList) => {
+    let outputList = [];
 
-    outputList.forEach(event => {
-      // If this is the creation of a project
-      if (
-        event.record_type === "moped_project" &&
-        event.operation_type === "INSERT"
-      ) {
-        // Remove that object from the array and add it back on at the end
-        outputList.splice(outputList.indexOf(event), 1);
-        outputList.push(event);
```

Comment:
this was code that went through all of the events and looked for the project creation event, then put it at the end of the array. 

I don't think its necessary, however I have found sometimes that seed data will show up in the activity log with project creation appearing after tags being added for example. I believe this would only happen in the seed data, since the events are sorted by creation date/time, and only when inserting seed data would all the events have the same time.